torch/_inductor/fx_passes/quantization.py

```diff
@@ -1778,8 +1778,9 @@ def _inner(match):
         for meta_value in [input_meta_value, weight_meta_value]:
             if (
                 meta_value is None
-                or meta_value.device.type != "cpu"
+                or (meta_value.device.type != "cpu" and meta_value.device.type != "xpu")
                 or meta_value.dim() != 4
+                or (meta_value.device.type == "xpu" and match.kwargs["groups"] != 1)
```

Comment:
@jansel , @desertfire , I'm wondering if you can help review the Inductor related changes.